lib/services/elasticsearch.js

```diff
@@ -1593,6 +1719,7 @@ class ElasticSearch extends Service {
             });
           }
         }
+        /* end critical code section */
 
         // @todo duplicated query to get documents body, mGet here and search in
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)